src/cljs/ataru/virkailija/editor/view.cljs

```diff
@@ -45,34 +44,34 @@
    "Uusi lomake"])
 
 (defn- copy-form []
-  (let [form-key  (subscribe [:state-query [:editor :selected-form-key]])
-        disabled? (reaction (nil? @form-key))]
+  (let [form-key  @(subscribe [:state-query [:editor :selected-form-key]])
+        disabled? (nil? form-key)]
     (fn []
       [:a.editor-form__control-button
        {:on-click (fn [event]
                     (.preventDefault event)
                     (dispatch [:editor/copy-form]))
-        :class    (if @disabled?
+        :class    (if disabled?
                     "editor-form__control-button--disabled"
                     "editor-form__control-button--enabled")}
        "Kopioi lomake"])))
 
 (defn- remove-form []
```

Comment:
Same comment as in above, "copy-form". The values remain static in render after this change.